test/servico/cst12Test.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package servico;
+
+import datamapper.PopulateDB;
+import datamapper.exceptions.NonexistentEntityException;
+import dominio.Aula;
+import dominio.Ausencia;
+import dominio.Professor;
+import java.text.ParseException;
+import java.util.LinkedList;
+import java.util.List;
+import modelo.AusenciaModel;
+import modelo.ProfessorModel;
+import org.joda.time.Interval;
+import org.junit.After;
+import org.junit.AfterClass;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ *
+ * @author GUSTAVO
+ */
+public class cst12Test {
+
+    public cst12Test() {

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://www.codacy.com/app/calebepb/substituicao/file/2770014455/issues/source?bid=3308597&fileBranchId=3353485#l33)